src/main/java/seedu/todo/commons/core/Config.java

```diff
@@ -37,28 +38,12 @@ public void setLogLevel(Level logLevel) {
         this.logLevel = logLevel;
     }
 
-    public String getUserPrefsFilePath() {
-        return userPrefsFilePath;
+    public String getDatabaseFilePath() {
+        return databaseFilePath;
     }
 
-    public void setUserPrefsFilePath(String userPrefsFilePath) {
-        this.userPrefsFilePath = userPrefsFilePath;
-    }
-
-    public String getAddressBookFilePath() {
-        return addressBookFilePath;
-    }
-
-    public void setAddressBookFilePath(String addressBookFilePath) {
-        this.addressBookFilePath = addressBookFilePath;
-    }
-
-    public String getAddressBookName() {
-        return addressBookName;
-    }
-
-    public void setAddressBookName(String addressBookName) {
-        this.addressBookName = addressBookName;
+    public void setDatabaseFilePath(String databaseFilePath) {
+        this.databaseFilePath = databaseFilePath;
```

Comment:
Oh crap, hmm... well all our problems will be solved if we force it to have a ".json" right? If we overwrite another json, it was probably an old db file. Won't break anything.

Confirmation can be a nice-to-have.